tests/tests_stratis.yml

```diff
@@ -66,94 +66,16 @@
             state: started
 
-        - name: Create one Stratis pool with one volume
-          include_role:
-            name: linux-system-roles.storage
-          vars:
-            storage_pools:
-              - name: foo
-                disks: "{{ unused_disks }}"
-                type: stratis
-                volumes:
-                  - name: test1
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location }}"
-
-        - name: Verify role results
-          include_tasks: verify-role-results.yml
-
-        - name: Repeat the previous invocation to verify idempotence
-          include_role:
-            name: linux-system-roles.storage
-          vars:
-            storage_pools:
-              - name: foo
-                disks: "{{ unused_disks }}"
-                type: stratis
-                volumes:
-                  - name: test1
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location }}"
-
-        - name: Verify role results
-          include_tasks: verify-role-results.yml
-
-        - name: Add second filesystem to the pool
-          include_role:
-            name: linux-system-roles.storage
-          vars:
-            storage_pools:
-              - name: foo
-                disks: "{{ unused_disks }}"
-                type: stratis
-                volumes:
-                  - name: test1
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location }}"
-                  - name: test2
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location_2 }}"
-
-        - name: Verify role results
-          include_tasks: verify-role-results.yml
-
-        - name: Clean up
-          include_role:
-            name: linux-system-roles.storage
-          vars:
-            storage_pools:
-              - name: foo
-                disks: "{{ unused_disks }}"
-                type: stratis
-                state: absent
-                volumes:
-                  - name: test1
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location }}"
-                    state: absent
-                  - name: test2
-                    size: "{{ volume_size }}"
-                    mount_point: "{{ mount_location_2 }}"
-                    state: absent
-
-        - name: Verify role results
-          include_tasks: verify-role-results.yml
-
-        - name: Run test only if blivet supports this functionality
-          when: ((is_fedora and blivet_pkg_version is version("3.10.0-1", ">=")) or
-                (is_rhel10 and blivet_pkg_version is version("3.10.0-1", ">=")) or
-                (is_rhel9 and blivet_pkg_version is version("3.6.0-15", ">=")))
+        - name: One pool/volume test
           block:
-            - name: Create encrypted Stratis pool
+            - name: Create one Stratis pool with one volume
```

Comment:
For reviewers: git and github make an utter mess of the diff view of "tests: Fix stratis block device clean up". What this does is to wrap the steps of a test into `block: ... always: Cleanup`. This changes the indendation, and due to the repetitive nature of the test the diff view locks onto" the wrong context. I didn't change any logic.